src/components/imagePanel/ImagePanel.tsx

```diff
@@ -2,9 +2,12 @@ import { Media, MediaDownloadType } from '@chili-publish/studio-sdk';
 import { convertToPreviewType } from '../../utils/mediaUtils';
 import ItemBrowser from '../itemBrowser/ItemBrowser';
 import { useVariablePanelContext } from '../../contexts/VariablePanelContext';
+import { useUiConfigContext } from '../../contexts/UiConfigContext';
 
 function ImagePanel() {
-    const { handleUpdateImage, currentVariableConnectorId } = useVariablePanelContext();
+    const { handleUpdateImage, currentVariableConnectorId, currentVariableId } = useVariablePanelContext();
+    const { onVariableBlur } = useUiConfigContext();
```

Comment:
and focus?

Reply:
The `ImagePanel` is basically the panel with the list of the assets, when we select an asset we blur, the `onVariableFocus` is in the `ImageVariable` component https://github.com/chili-publish/studio-ui/blob/8ecd8778eaeb9076b555ce87d46e8649a660eb69/src/components/variablesComponents/imageVariable/ImageVariable.tsx#L70